src/response/records.ts

```diff
@@ -361,6 +361,35 @@ export interface Disposition {
   readonly ruleLabel?: string;
 }
 
+/**
+ * This object describes one of the reasons for the multiplier.
+ */
+export interface Reason {
+  /**
+   * The machine-readable code for the reason.
```

Comment:
A link to the dev site with the list of codes might be nice, but probably isn't essential.